modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceQuickAddToCart.testcase

```diff
@@ -1631,6 +1631,69 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10475. User can paste text on quick add to cart search bar"
+	@priority = 3
+	test CanPasteTextOnQuickAddToCartSearchBar {
+		property portal.acceptance = "false";
+
+		task ("Given a business account with a user without any roles") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "testuser@liferay.com",
+				userFirstName = "Test",
+				userLastName = "User",
+				userScreenName = "testuser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And the user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "testuser@liferay.com");
+		}
```

Comment:
@jaydawu can you use a buyer user instead of this one for the test?
If the test you're writing does assertion on the store-front, I suggest to always use a buyer user. Thanks!